Dockerfile.amd64

```diff
@@ -31,6 +31,38 @@ ADD k8s-install/scripts/calico.conf.default /calico.conf.tmp
 RUN microdnf install hostname && mkdir /licenses
 COPY LICENSE /licenses
 
+RUN microdnf update
+
+# Remove packages currently known to have vulnerabilities and dependencies
+# Remove packages deemed known to be unsafe by threat defense team
+RUN microdnf remove cyrus-sasl-lib \
```

Comment:
Does the order matter here? I saw someplace else that it mattered. Curious if this is the case here.

Reply:
It does not appear to matter. microdnf seems to be smart enough to understand (or sort itself based on dependencies).